vendor.sh

```diff
@@ -5,8 +5,27 @@
 set -e
 
 function download_sitter() {
-    rm -rf vendor
+    rm -rf vendor;
```

Comment:
do we need this change?

Reply:
In ignores a potential error. I can remove this line as well.